README.md

```diff
@@ -56,7 +62,88 @@ If you have a Snyk token, add that as an environment variable:
 - Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables.
 - Set the variable name to _EXHORT_SNYK_TOKEN_, and copy-and-paste your Snyk token into the value field.
 
-#### Option I- As a build step
+
+#### General Configuration
+ Click <em>Manage Jenkins</em>. Click <em>System</em>, and scroll down to <em>Global properties/Environment Variables</em>. Here you can configure the following settings:
+ - name: _EXHORT_DEBUG_, Value: true , Description: Will invoke the analysis in verbose mode and will print a lot of useful logs to job output console - good for debugging, Default value is false.
+
+
+ - name: _EXHORT_DEV_MODE_, value: true, Description: Will invoke the Analysis on Staging Instance Of EXHORT Service, Default: false ( EXHORT Production Instance)
+
+
+ - name: _HIGHEST_ALLOWED_VULN_SEVERITY_, Possible values: [`LOW`,`MEDIUM`,`HIGH`,`CRITICAL`], Description: will determine what is the highest allowed Severity of a vulnerability found for a given package/dependency in the analysis, for the analysis to be considered Successful(RC=0) and not Vulnerable(RC=2), Default value is `MEDIUM`
+
+#### Python Pipeline Configuration
+ For Python PIP packages, you can use the specific Python and PIP binaries during the invocation of the analysis. You can also specify these binaries elsewhere in your pipeline jobs, such as a stage environment, or another agent or node. Red Hat Dependency Analytics gives you maximum flexibility with the Python and PIP versions. You do not have to enforce the user to install different Python and PIP versions just to adapt it to the exact `requirements.txt` list of package versions. Python is very sensitive to versioning, for each Python version, there is a limited range of supported versions for a package.
+ There are two environment variables:
+  1. _EXHORT_PIP_FREEZE_
+  2. _EXHORT_PIP_SHOW_ 
+ 
+
+ This feature enable you to use python on different agent ( for example, container image of python containing the desired python version), install the input requirements.txt using pip in the container image, and then invoke `pip freeze --all` and `pip show [List_Of_Packages]`, and save the output of these to a workspace files,
+ And then set the above variables with these output after encoding them using base64 encoding.
```

Comment:
This sentence can be added to the end of the paragraph.  For example, using my previous suggestion as the paragraph:

... then you can use the following commands to generate the output for your workspace file: pip freeze --all and pip show <list_of_packages>. Next, run `base64` to encode the output from these commands, and set the `EXHORT_PIP_FREEZE` and `EXHORT_PIP_SHOW` environment variables with that encoded output, respectively.

Reply:
Done, Thanks.